Source/WebCore/platform/graphics/Damage.h

```diff
@@ -75,6 +75,7 @@ class Damage {
     // May return both empty and overlapping rects.
     ALWAYS_INLINE const Rects& rects() const { return m_rects; }
     ALWAYS_INLINE bool isEmpty() const  { return m_rects.isEmpty(); }
+    ALWAYS_INLINE Mode mode() const { return m_mode; }
 
     void makeFull(const IntSize& size)
```

Comment:
Side note: I dislike the name `makeFull` -- doesn't trigger any bells :-)
`extendToCoverWholeArea` would be more descriptive.

Reply:
There's a Full mode, so this makes the damage Full. Before there was a setMode but we don't really need to support changing from any mode to another, only make it full.